spec/features/mailer_settings_spec.rb

```diff
@@ -0,0 +1,54 @@
+RSpec.describe "Configuring mailer settings" do
+  let(:code) { <<-CODE.strip_heredoc
+
+        # Mailer config
+        config.action_mailer.perform_deliveries = true
+        config.action_mailer.default_url_options = { host: 'something'}
+        config.action_mailer.asset_host = 'something'
+        config.action_mailer.smtp_settings = {
+          :address   => "smtp.mandrillapp.com",
```

Comment:
our defaults here have changed for sure and I dont think writing tests for this is helpful, it just makes things harder to change